HomeWork_3/src/CheckResult.java

```diff
@@ -25,29 +25,27 @@ public static String check(double number) {
      * @return False if zero!=0.
      */
     public static boolean checkZero(double zero){
-         if(zero == 0) {	
-           System.out.println(Helper.DIVISION_BY_ZERO);
+         if(!(Double.isInfinite(zero))) {	
```

Comment:
This check looks error-prone - zero is always finite I could not imagine what are you doing here